Philippewanner feature iot rule action lambda python cdk ts (#2510)
aws-samples/serverless-patterns: iot-lambda-pub-receiver-cdk/README.md

# AWS Lambda to AWS IoT Core

> This pattern deploys one publisher AWS Lambda, which publishes a message in AWS IoT Core topic. The topic is watch by a rule which triggers an action when the condition is met. The action calls a AWS Lambda.

Reviewer: ```suggestion
This pattern deploys an AWS Lambda function, which publishes a message to an AWS IoT Core topic. The topic is watched by a rule which will trigger an action when the condition is met. The action calls an AWS Lambda function.
```

> Learn more about this pattern at Serverless Land Patterns: << Add the live URL here >>

Reviewer: Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/iot-lambda-pub-receiver-cdk
